src/config/myanmar/layers.json

```diff
@@ -1197,7 +1197,10 @@
       "datetime_field": "date_time",
       "geom_field": "location",
       "measure_field": "number",
-      "kobo_url": "https://kobo.humanitarianresponse.info/api/v2/assets.json"
+      "kobo_url": "https://kobo.humanitarianresponse.info/api/v2/assets.json",
+      "filters": {
+        "status": "Approved"
+      }
```

Comment:
Tested Myanmar `Kobo data` and since you are filtering by `status` to equal `Approved` we should only see approved `point_data` only. I still am seeing `On hold`.

Reply:
I have added `status` into `feature_info_props` to visual see if on the map I will only see `Approved` and still I had `On hold`. Correct me if my assumption on `filters` is wrong.

Here is the screenshot for clarity
<img width="1552" alt="Screen Shot 2021-11-02 at 6 24 33 PM" src="https://user-images.githubusercontent.com/1290048/139915012-7240d94f-39c7-4cfb-b925-0067cb958da5.png">